cfme/infrastructure/virtual_machines.py

```diff
@@ -1020,10 +1124,8 @@ def step(self, *args, **kwargs):
                 datacenter = self.obj.datacenter
         else:
             raise DestinationNotFound("the destination isn't found")
-        accordion.tree('VMs & Templates', 'All VMs & Templates', provider, datacenter)
-
-    def resetter(self, *args, **kwargs):
-        reset_page()
+        self.view.sidebar.vmstemplates.tree.click_path('VMs & Templates',
```

Comment:
This click_path looks redundant, given that you're calling it from sidebar.vmstemplates.tree - which should be the tree in the 'VMs & Templates' accordion already.

Note how in cloud.instances.AllForProvider.step, the first argument to 'sidebar.instances_by_provider.tree.click_path' is 'Instances by Provider', which is the folder label within the accordion. 

Believe your first argument here should be 'All VMs & Templates`, since its being given against a specific accordion .